### PR TITLE
Remove redundant getters

### PR DESCRIPTION
**Pull request by rabi**
We don't need those for public fields.

### CODE REVIEW

File: controllers/core/openstackcontrolplane_controller.go [hunk 1]
```diff
@@ -47,26 +47,6 @@ import (
...
-
-// GetLogger -
-func (r *OpenStackControlPlaneReconciler) GetLogger() logr.Logger {
-	return r.Log
-}
-
-// GetScheme -
-func (r *OpenStackControlPlaneReconciler) GetScheme() *runtime.Scheme {
-	return r.Scheme
-}
```
Comment (abays) [inline controllers/core/openstackcontrolplane_controller.go:68]: @stuggi Are these just essentially copy-pastes from our old pattern in OSPdO?  Just wondering if we still use this pattern somewhere in any of the new operators.

Comment (stuggi) [reply controllers/core/openstackcontrolplane_controller.go:68]: yes, those are left overs from the older ReconcilerCommon and are now no longer needed with the newer Helper

**Comment by dprince** [comment]
yes, lgtm but it needs a rebase now that transport URL lives in the infra-operator

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**